### PR TITLE
more typical dir structure around workspace crates

### PR DESCRIPTION
**Pull request by zeeshanlakhani**
## Summary

Match directory structure akin to libs like tokio/tracing/etc, while still matching the names of the crates on crates.io. This looks like a big PR, but is just names moving around.

- [x] Branched from https://github.com/wnfs-wg/rs-wnfs/pull/72, so that should merge first.
- [x] Will rebase after #68 and #70

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -276,24 +269,9 @@ This library recommends using [pre-commit](https://pre-commit.com/) for running
 
 ## Getting Help
 
-For usage questions, usecases, or issues reach out to us in our [Discord webnative-fs channel](https://discord.com/channels/478735028319158273/877623827331428403).
+For usage questions, usecases, or issues reach out to us in our [Discord webnative-fs channel](https://discord.gg/YbT6x7Wkvk).
 We would be happy to try to answer your question or try opening a new issue on Github.
 
-## External Resources
```
Comment (zeeshanlakhani) [inline README.md:282]: moving this to profile for group: https://github.com/wnfs-wg

File: wnfs-wasm/Cargo.toml [hunk 2]
```diff
@@ -12,7 +12,7 @@ categories = [
 license = "Apache-2.0"
 readme = "README.md"
 edition = "2021"
-repository = "https://github.com/WebNativeFileSystem/rs-wnfs/tree/main/crates/wasm"
+repository = "https://github.com/WebNativeFileSystem/rs-wnfs/tree/main/wnfs-wasm"
```
Comment (appcypher) [inline wnfs-wasm/Cargo.toml:15]: We should also update this to `wnfs-wg`
```suggestion
repository = "https://github.com/wnfs-wg/rs-wnfs/tree/main/wnfs-wasm"
```

File: wnfs/Cargo.toml [hunk 3]
```diff
@@ -12,7 +12,7 @@ categories = [
 license = "Apache-2.0"
 readme = "README.md"
 edition = "2021"
-repository = "https://github.com/WebNativeFileSystem/rs-wnfs/tree/main/crates/fs"
+repository = "https://github.com/WebNativeFileSystem/rs-wnfs/tree/main/wnfs"
```
Comment (appcypher) [inline wnfs/Cargo.toml:15]: Same here.
```suggestion
repository = "https://github.com/wnfs-wg/rs-wnfs/tree/main/wnfs"
```

File: wnfs/README.md [hunk 4]
```diff
@@ -29,8 +29,6 @@
 
 <div align="center"><sub>:warning: Work in progress :warning:</sub></div>
 
-##
-
```
Comment (appcypher) [inline wnfs/README.md:33]: This was added to visually separate the two sections. Can it be added back?

Comment (zeeshanlakhani) [reply wnfs/README.md:33]: ah, got it, i was reading in markdown and confused, yep

File: wnfs/README.md [hunk 5]
```diff
@@ -7,7 +7,7 @@
```
Comment (appcypher) [inline wnfs/README.md:7]: Minor links and texts fixes:
- `crates/fs/examples/` -> `examples/`
- `../../assets/logo.png` -> `../assets/logo.png`

**Review by appcypher — APPROVED** [review]
<div align="center">
    <a href="#" target="_blank">
        </img>
    </a>
</div>

<h3 align="center">LGTM! 🎉 </h3>

**Pull request merged**